### PR TITLE
feat(home-page): create demo using nivels production data

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

**Review by kibo-sushant — APPROVED** [review]

File: components/home/LargeBanner/LargeBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  const router = useRouter()
+
+  return (
+    <>
+      {bannerProps && (
+        <Card sx={styles.contentStyle}>
+          <CardMedia sx={styles.cardMediaStyle}>
+            <KiboImage
+              src={backgroundImageUrl}
+              alt={backgroundImageUrl ? 'abc' : 'product-image-alt'}
```
Comment (kibo-chandradeeptalaha) [inline components/home/LargeBanner/LargeBanner.tsx]: What is abc here?

File: components/layout/AppHeader/Common/KiboHeader.tsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ const headerActionAreaStyles = {
   },
   logoWrapper: {
     order: 0,
-    top: '-27px',
+    // top: '-27px',
   },
```
Comment (kibo-chandradeeptalaha) [inline components/layout/AppHeader/Common/KiboHeader.tsx:95]: Remove commented code.

File: cms/components/CmsFeaturedCategories/CmsFeaturedCategories.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  component: CmsFeaturedCategories,
+} as ComponentMeta<typeof CmsFeaturedCategories>
+
+const Template: ComponentStory<typeof CmsFeaturedCategories> = (args) => (
+  <CmsFeaturedCategories {...args} />
+)
+
+export const Common = Template.bind({})
+
+Common.args = {
```
Comment (kibo-chandradeeptalaha) [inline cms/components/CmsFeaturedCategories/CmsFeaturedCategories.stories.tsx:19]: Should we use separate mock files for this, as it is used in another file as well?

File: cms/components/CmsVehicleInnovation/CmsVehicleInnovation.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+export const Common = Template.bind({})
+
+const vehicleInnovationProps = {
+  title: '50 Years of Specialty Vehicle Innovation',
+  subtitle: "It's What Drives Us",
+  backgroundImageUrl: InnovationBackgroundImage,
+  mobileViewLinkTitle: 'Learn more about us',
+  mobileViewLinkUrl: '/',
+  footerChildrens: [
+    {
```
Comment (kibo-chandradeeptalaha) [inline cms/components/CmsVehicleInnovation/CmsVehicleInnovation.stories.tsx]: Should we use separate mock files for this, as it is used in another file as well?

File: components/home/LargeBanner/LargeBanner.tsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+              layout="fill"
+              objectFit="cover"
+              data-testid="product-image"
+            />
+
+            <CardContent sx={styles.cardStyle}>
+              <Typography
+                variant="h2"
+                sx={styles.nameStyle}
+                style={{ fontSize: mobileView ? '18px' : '36px' }}
```
Comment (kibo-chandradeeptalaha) [inline components/home/LargeBanner/LargeBanner.tsx]: use rem instead px, try to use it from theme.

File: components/product/ProductOptionSelect/ProductOptionSelect.spec.js [hunk 6]
```diff
@@ -3,8 +3,8 @@ import { render, screen, within, fireEvent } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 
 import '@testing-library/jest-dom'
-import { productOptionSelectValuesMock } from '../../../__mocks__/stories/productOptionSelectMock'
 import * as stories from './ProductOptionSelect.stories' // import all stories from the stories file
+import { productOptionSelectValuesMock } from '../../../__mocks__/stories/productOptionSelectMock'
```
Comment (kibo-chandradeeptalaha) [inline components/product/ProductOptionSelect/ProductOptionSelect.spec.js:7]: use absolute path.

File: components/product/ProductOptionSelect/ProductOptionSelect.stories.tsx [hunk 7]
```diff
@@ -2,8 +2,8 @@ import React from 'react'
 
 import { ComponentStory, ComponentMeta } from '@storybook/react'
 
-import { productOptionSelectValuesMock } from '../../../__mocks__/stories/productOptionSelectMock'
 import ProductOptionSelect, { ProductOptionSelectProps } from './ProductOptionSelect'
+import { productOptionSelectValuesMock } from '../../../__mocks__/stories/productOptionSelectMock'
```
Comment (kibo-chandradeeptalaha) [inline components/product/ProductOptionSelect/ProductOptionSelect.stories.tsx:6]: use absolute path.

File: components/common/PasswordValidation/PasswordValidation.spec.tsx [hunk 8]
```diff
@@ -3,8 +3,8 @@ import React from 'react'
 import { composeStories } from '@storybook/testing-react'
 import { render, screen } from '@testing-library/react'
 
-import theme from '../../../styles/theme'
 import * as stories from './PasswordValidation.stories'
+import theme from '../../../styles/theme'
```
Comment (kibo-chandradeeptalaha) [inline components/common/PasswordValidation/PasswordValidation.spec.tsx:7]: import theme from '@/styles/theme'
Check in other files as well.

File: components/core/ImageGallery/ImageGallery.stories.tsx [hunk 9]
```diff
@@ -2,8 +2,8 @@ import React from 'react'
 
 import { ComponentStory, ComponentMeta } from '@storybook/react'
 
-import { imageGalleryData } from '../../../__mocks__/stories/imageGalleryDataMock'
 import ImageGallery from './ImageGallery'
+import { imageGalleryData } from '../../../__mocks__/stories/imageGalleryDataMock'
```
Comment (kibo-chandradeeptalaha) [inline components/core/ImageGallery/ImageGallery.stories.tsx:6]: use absolute path.

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]

File: components/home/LargeBanner/LargeBanner.tsx (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply components/home/LargeBanner/LargeBanner.tsx]: Resolved

File: components/layout/AppHeader/Common/KiboHeader.tsx (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply components/layout/AppHeader/Common/KiboHeader.tsx:95]: Resolved

File: cms/components/CmsFeaturedCategories/CmsFeaturedCategories.stories.tsx (hunk 3 above)
Comment (kibo-geetanshuchhabra) [reply cms/components/CmsFeaturedCategories/CmsFeaturedCategories.stories.tsx:19]: Resolved

File: cms/components/CmsVehicleInnovation/CmsVehicleInnovation.stories.tsx (hunk 4 above)
Comment (kibo-geetanshuchhabra) [reply cms/components/CmsVehicleInnovation/CmsVehicleInnovation.stories.tsx]: Resolved

File: components/home/LargeBanner/LargeBanner.tsx (hunk 5 above)
Comment (kibo-geetanshuchhabra) [reply components/home/LargeBanner/LargeBanner.tsx]: Resolved

File: components/core/ImageGallery/ImageGallery.stories.tsx (hunk 9 above)
Comment (kibo-geetanshuchhabra) [reply components/core/ImageGallery/ImageGallery.stories.tsx:6]: This is a Fork Branch. All these changes will have to be done in the nextjs-storefront branch.

File: components/common/PasswordValidation/PasswordValidation.spec.tsx (hunk 8 above)
Comment (kibo-geetanshuchhabra) [reply components/common/PasswordValidation/PasswordValidation.spec.tsx:7]: This is a Fork Branch. All these changes will have to be done in the nextjs-storefront branch.

File: components/product/ProductOptionSelect/ProductOptionSelect.stories.tsx (hunk 7 above)
Comment (kibo-geetanshuchhabra) [reply components/product/ProductOptionSelect/ProductOptionSelect.stories.tsx:6]: This is a Fork Branch. All these changes will have to be done in the nextjs-storefront branch.

File: components/product/ProductOptionSelect/ProductOptionSelect.spec.js (hunk 6 above)
Comment (kibo-geetanshuchhabra) [reply components/product/ProductOptionSelect/ProductOptionSelect.spec.js:7]: This is a Fork Branch. All these changes will have to be done in the nextjs-storefront branch.

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**